test_conformance/extensions/cl_khr_command_buffer/cl_khr_command_buffer_mutable_dispatch/mutable_command_interleave.cpp

```diff
@@ -0,0 +1,808 @@
+            "    pBufs->pB[i]++;\n"
+            "    pBufs->pC[i]++;\n"
+            "}\n"
+        };
+
+        clProgramWrapper program = clCreateProgramWithSource(
+            context, 1, &set_kernel_exec_info_svm_ptrs_kernel, nullptr, &error);
+        test_error(error, "Unable to create program");
+
+        error = clBuildProgram(program, 1, &device, nullptr, nullptr, nullptr);
+        test_error(error, "Unable to build program");
+
+        kernel = clCreateKernel(program, "set_kernel_exec_info_test", &error);
+        test_error(error, "Unable to create kernel");
+
+        cl_command_buffer_properties_khr properties[3] = {
+            CL_COMMAND_BUFFER_FLAGS_KHR, 0, 0
+        };
+        properties[1] = CL_COMMAND_BUFFER_MUTABLE_KHR;
+
+        mutable_command_buffer =
+            clCreateCommandBufferKHR(1, &queue, properties, &error);
+        test_error(error, "clCreateCommandBufferKHR failed");
+
+        return 0;
+    }
+
+    cl_int Run() override
+    {
+        cl_ndrange_kernel_command_properties_khr props[] = {
+            CL_MUTABLE_DISPATCH_UPDATABLE_FIELDS_KHR,
+            CL_MUTABLE_DISPATCH_ARGUMENTS_KHR, 0
+        };
+
+        size_t s = num_elements * sizeof(int);
+        int *pA = (int *)clSVMAlloc(context, CL_MEM_READ_WRITE, s, 0);
+        int *pB = (int *)clSVMAlloc(context, CL_MEM_READ_WRITE, s, 0);
+        int *pC = (int *)clSVMAlloc(context, CL_MEM_READ_WRITE, s, 0);
+        BufPtrs *pBuf = (BufPtrs *)clSVMAlloc(context, CL_MEM_READ_WRITE,
+                                              sizeof(BufPtrs), 0);
+
+        pBuf->pA = pA;
+        pBuf->pB = pB;
+        pBuf->pC = pC;
+
+        cl_int error = clSetKernelArgSVMPointer(kernel, 0, pBuf);
+        test_error(error, "clSetKernelArg failed");
+
+        error = clSetKernelExecInfo(kernel, CL_KERNEL_EXEC_INFO_SVM_PTRS,
+                                    sizeof(BufPtrs), pBuf);
+        test_error(error, "clSetKernelExecInfo failed");
+
+        error = clCommandNDRangeKernelKHR(
+            mutable_command_buffer, nullptr, props, kernel, 1, nullptr,
+            &global_work_size, nullptr, 0, nullptr, nullptr, &command);
+        test_error(error, "clCommandNDRangeKernelKHR failed");
+
+        error = clFinalizeCommandBufferKHR(mutable_command_buffer);
+        test_error(error, "clFinalizeCommandBufferKHR failed");
+
+        error = clEnqueueCommandBufferKHR(0, nullptr, mutable_command_buffer, 0,
+                                          nullptr, nullptr);
+        test_error(error, "clEnqueueCommandBufferKHR failed");
+
+        cl_mutable_dispatch_exec_info_khr exec_info_list{
+            CL_KERNEL_EXEC_INFO_SVM_PTRS, sizeof(BufPtrs), pBuf
+        };
+
+
+        cl_mutable_dispatch_config_khr dispatch_config{
+            CL_STRUCTURE_TYPE_MUTABLE_DISPATCH_CONFIG_KHR,
+            nullptr,
+            command,
+            0 /* num_args */,
+            0 /* num_svm_arg */,
+            0 /* num_exec_infos */,
```

Comment:
this should be getting set to 1 since you're passing 1 element to`exec_info_list`